src/braket/circuits/gates.py

```diff
@@ -2121,6 +2204,28 @@ def get_angle(self, **kwargs):
     return type(self)(angle=new_angle)
 
 
+def get_angles(self, **kwargs):
```

Comment:
Does this need to be public? Honestly, I'm not sure `get_angle` and `format_complex` need to be public either, but we need to keep it that way for backwards compatibility.